### PR TITLE
[ADD] oxford_asl descriptor

### PR DESCRIPTION
**Pull request by natacha-beck**

### CODE REVIEW

File: cbrain_task_descriptors/oxford_asl.json [hunk 1]
```diff
@@ -0,0 +1,454 @@
...
+            "type": "File",
+            "list": false,
+            "optional": false,
+            "id": "asl_data",
+            "name": "Asl data"
+        },
+        {
+            "command-line-flag": "-o",
+            "description": "Output directory name",
+            "value-key": "[OUTPUT_DIR]",
```
Comment (prioux) [inline cbrain_task_descriptors/oxford_asl.json:27]: What checks are made on this? What prevents a user from entering "/home/myusername/.ssh" and trashing a system directory unrelated to the task?

File: cbrain_task_descriptors/oxford_asl.json [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+        "path-template-stripped-extensions": [".nii.gz", ".nii"]
+        }
+    ],
+    "tags": {
+        "domain": [
+            "neuroinformatics",
+            "mri"
+        ]
+    },
+    "description": "oxford_asl is part of BASIL (https://asl-docs.readthedocs.io/en/latest/oxford_asl_userguide.html)"
```
Comment (prioux) [inline cbrain_task_descriptors/oxford_asl.json:453]: Why is the description at the bottom?

File: cbrain_task_descriptors/oxford_asl.json [hunk 3]
```diff
@@ -0,0 +1,454 @@
...
+}
```
Comment (prioux) [inline cbrain_task_descriptors/oxford_asl.json:454]: The descriptor is missing the `custom[]` structure with the cbrain info

File: cbrain_task_descriptors/oxford_asl.json (hunk 1 above)
Comment (natacha-beck) [reply cbrain_task_descriptors/oxford_asl.json:28]: CBRAIN do some sanity check.

**Comment by natacha-beck** [comment]
Add oxford_asl descriptor for oxford_asl in FSL (BASIL).

**Pull request merged**